### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by houtarchat-cyber** [houtarchat-cyber/EWT-get-answer#4] [General]
## What's Changed
* Bump @types/node from 14.18.26 to 18.7.14 by @dependabot in https://github.com/houtarchat-cyber/EWT-get-answer/pull/1
* Bump @types/node from 18.7.14 to 18.7.15 by @dependabot in https://github.com/houtarchat-cyber/EWT-get-answer/pull/3
* Bump eslint from 8.22.0 to 8.23.0 by @dependabot in https://github.com/houtarchat-cyber/EWT-get-answer/pull/2

## New Contributors
* @dependabot made their first contribution in https://github.com/houtarchat-cyber/EWT-get-answer/pull/1

**Full Changelog**: https://github.com/houtarchat-cyber/EWT-get-answer/compare/v0.2.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/houtarchat-cyber/EWT-get-answer/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by hOawaOh** [comment]
页面多少有点碍事，把窗口变小放在右下角.
const answerShower = window.open('', '_blank', 'width=400,height=200');
            if (answerShower) {
               answerShower.document.body.innerHTML = answers;
               answerShower.moveTo(window.screen.width - answerShower.outerWidth, window.screen.height - answerShower.outerHeight);
            }
